src/CustomSearch.jsx

```diff
@@ -0,0 +1,35 @@
+import React, {useState, useRef, useEffect} from "react";
+
+function CustomSearch() {
+  const [adressForm, setAdressform] = React.useState('')
+  const [adressCoordinates, setAdresscoordinates] = React.useState('') 
+
+  function handleSubmit(event) {
+    event.preventDefault()
+    setAdressform(event.target.suggest.value)
+    
+    var myGeocoder = ymaps.geocode(event.target.suggest.value);
```

Comment:
Don't use `var`! Use `const` when you don't want to reassign data, or when you use any kind of objects (arrays, maps, sets, etc.) and `let` when you plan to reassign data. `var`'s behavior is much more unpredictable than these alternatives

Reply:
done